docs/pages/connect-your-client/web-ui.mdx

```diff
@@ -31,3 +31,22 @@ You must have the `join_sessions` allow policy in a role you've been assigned to
 For information about how to configure the `join_sessions` allow policy and participant modes for a role, see 
 [Configure an allow policy](../access-controls/guides/moderated-sessions.mdx#configure-an-allow-policy).
 
+## Secure Copy (SCP) files with the Web UI
+
+Teleport provides two methods to copy local files to a remote system through the web interface.
+
+### Teleport UI
+
+You can use the upload functionality:
+
+![Teleport UI](../../img/download-files.png)
+
+You can also upload files from a system using drag and drop:
+
+![drag and drop](../../img/upload-ui.png)
+
+### Teleport Connect
+
+Teleport Connect is a graphical user interface for Teleport that runs on Mac, Linux and Windows. The interface also allows you to upload local files to a remote server.
```

Comment:
I'd also point out that the UI for that is accessible through the arrows in the top left of an ssh tab. Same for the Web UI. I remember when I was first using Teleport, I totally missed that upload/download is available because I totally missed those arrows!

Reply:
Oh nice! I didn't catch that. Thanks, I will add that as well